### DISCUSSION TITLE
Randomly got suspended with no reason or notice

### DISCUSSION THREAD

**Discussion by ghost** [community/community#130257] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I was looking at different tags for my repo when suddenly, I was logged out. When I tried to sign in again I got "Account suspended. Access to your account has been suspended due to a violation of our Terms of Service.",
This may have been because I was sending a significant number of HTTP requests because I was looking at the popularity of them.
so, of course, I sent an appeal right away. (Ticket: 2857013)

I have read the TOS and I couldn't find anything that I did against it.

I never got an email saying why it was suspended or received any follow up emails to my appeal.

Can you please prioritize the appeal or do something about it
My original account is `github.com/Py-mon`
I use github everyday without it, is hurting my academics.

Thank you.

**Comment by ebndev** [comment]
Hi @Pymon02,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
